com.github.thetumultuousunicornofdarkness.cpu-x.yaml

```diff
@@ -0,0 +1,67 @@
+app-id: com.github.thetumultuousunicornofdarkness.cpu-x
```

Comment:
app-id should by `io.github.*` as per the documentation.

Reply:
Done in io.github.thetumultuousunicornofdarkness.cpu-x.